docs/reference-guides/providers/docker_kubeadm.md

```diff
@@ -0,0 +1,58 @@
+---
+sidebar_position: 0
+---
+
+# Create & import a cluster using CAPI provider for Docker
+
+This section will guide you through creating a cluster and importing it into Rancher Manager using kubectl.
+
+## Prerequisites
+
+- Rancher Manager cluster with Rancher Turtles installed
+- Cluster API providers installed for your scenario - we'll be using the Docker infrastructure and Kubeadm bootstrap/control plane providers in these instructions - you can find a guide on how to install a provider using the `CAPIProvider` resource [here](../../tasks/capi-operator/basic_cluster_api_provider_installation.md)
+- **clusterctl** CLI - see the [releases](https://github.com/kubernetes-sigs/cluster-api/releases)
```

Comment:
Do you think it is good idea to have a list of Prerequisites which are common to most of the providers defined once somehere like intro page under providers and referring to it from provider specific pages instead of duplicating it each time? 

Or another idea would to have something similar to what CAPI quick start page has: https://cluster-api.sigs.k8s.io/user/quick-start#initialization-for-common-providers where they use [`tabs`](https://github.com/kubernetes-sigs/cluster-api/blob/1e6896eb086f59ded5ff745a75666987b48e9b6e/docs/book/src/user/quick-start.md?plain=1#L284)